DataFormats/Luminosity/interface/PixelClusterCountsPerEvent.h

```diff
@@ -0,0 +1,46 @@
+#ifndef DataFormats_PixelClusterCountsPerEvent_h
+#define DataFormats_PixelClusterCountsPerEvent_h
+/** \class reco::PixelClusterCountsPerEvent
+ *  
+ * Reconstructed PixelClusterCountsPerEvent object that will contain the moduleID, bxID, and counts per event.
+ *
+ * \authors: Sam Higginbotham (shigginb@cern.ch), Chris Palmer (capalmer@cern.ch), Attila Radl (attila.radl@cern.ch) 
+ * 
+ *
+ */
+#include <algorithm>
+#include <string>
+#include <sstream>
+#include <iostream>
+#include <vector>
+
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+
+#include "DataFormats/Luminosity/interface/LumiConstants.h"
+
+namespace reco {
+  class PixelClusterCountsPerEvent {
```

Comment:
It's a bit unfortunate that PixelClusterCounts is already used for per lumi data.
"PerEvent" sounds like this has accumulated statistics over many events.
Perhaps "PixelClusterCountsInEvent" is a better choice.